src/main/java/mate/academy/springbootintro/repository/BookRepository.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springbootintro.repository;
+
+import mate.academy.springbootintro.model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
implement logic of repo by yourself
```suggestion
public interface BookRepository {
```